### ISSUE TITLE
[FR] Default to new recorder interface

### ISSUE THREAD

**Issue by wenfangdu** [NickeManarin/ScreenToGif#969]
Currently, after a new STG installation, the recorder interface defaults to old, can we switch to new for the default? Since the new recorder interface has been out for quite some time now, and I haven't encountered many problems with it.

**Comment by NickeManarin** [comment]
Yes, that's the plan for v3. I just need to improve a bit the UX of the recorder.
